components/microservices/microservicesPanel.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import { CustomLink } from "../customLink";
+
+type MicroservicesPanelProps = {
+  title?: string;
+  description: string;
+  url?: string;
+};
+
+const MicroservicesPanel = ({
+  title,
+  description,
+  url,
+}: MicroservicesPanelProps) => {
+  return (
+    <>
+      <div className="border-2 bg-gray-100 p-5">
+        {!!title && <strong>{title}</strong>}
+        <p className="pt-5">{description}</p>
+        {!!url && (
+          <div className="flex justify-center pt-5">
+            <CustomLink
+              className="unstyled flex h-12 w-80 shrink-0 cursor-pointer items-center justify-center rounded bg-ssw-red px-5 uppercase text-white hover:opacity-70 max-sm:my-5 sm:w-fit"
+              href={url}
+            >
+              Learn more
```

Comment:
`Learn more` as link text is not a good for SEO

https://www.seomechanic.com/optimize-cta-better-alternatives-to-learn-more/

Reply:
it is now configurable